services/reference/unichain/json-rpc-methods/_eth_unichain-call-parameters.mdx

```diff
@@ -0,0 +1,15 @@
+- `from`: 20 bytes [_required_] Address the transaction is sent from.
+- `to`: 20 bytes - Address the transaction is directed to.
+- `gas`: Hexadecimal value of the gas provided for the transaction execution. `eth_call` consumes zero gas, but this parameter may be needed by some executions.
+- `gasPrice`: Hexadecimal value of the `gasPrice` used for each paid gas.
+- `maxPriorityFeePerGas`: Maximum fee, in wei, the sender is willing to pay per gas above the base fee. See [EIP-1559 transactions](../../concepts/transaction-types.md#eip-1559-transactions).
+- `maxFeePerGas`: Maximum total fee (base fee + priority fee), in wei, the sender is willing to pay per gas. See [EIP-1559 transactions](../../concepts/transaction-types.md#eip-1559-transactions).
+- `value`: Hexadecimal of the value sent with this transaction.
+- `data`: Hash of the method signature and encoded parameters. See [Ethereum contract ABI specification](https://docs.soliditylang.org/en/latest/abi-spec.html).
+- `blockParameter`:  (string) [_required_] A hexadecimal block number, or one of the tags `latest`, `earliest`, `pending` or `finalized`. See the [default block parameter](https://ethereum.org/en/developers/docs/apis/json-rpc/#default-block).
+
+	:::warning
+	`safe` isn't supported. Use `finalized` instead. 
+	:::
+
+
```

Comment:
```suggestion
  :::warning
  `safe` isn't supported. Use `finalized` instead. 
  :::
	
```

Remove line break to prevent list from rendering line breaks between every item.